reri_err_reporting.adoc

```diff
@@ -414,20 +442,22 @@ the hardware unit.
   {bits: 1,  name: 'c'},
```

Comment:
I was expecting to see a set of bits associated with the error event to provide details about the specifics of the event and subsume bits like `c` and `scrub`.

Reply:
A set of bits in the `status_i` register are designated for future standard use and may be defined by future standard extensions for additional qualifiers.